src/components/event/experimental-api/ExperimentalAPIEventStore.ts

```diff
@@ -25,13 +27,35 @@ export class ExperimentalAPIEventStore {
 
 	@observable scrolledIndex: null | Number = null;
 
-	constructor() {
+	events: EventStoreDefaultStateType;
+
+	constructor(events: EventStoreDefaultStateType) {
+		this.events = events;
 		this.getRootIds();
+		this.loadEvent((events as any)?.selectedEventId);
 	}
 
+	@action
+	loadEvent = async (eventId: string) => {
+		if (eventId) {
+			const data = await this.fetchEvent(eventId);
+			const firstChild = JSON.parse(JSON.stringify(data));
+			const parents = await eventHttpApi.getEventParents(firstChild.eventId);
```

Comment:
look's like `getEventParents` requires firstParentId as an argument. which probably a bit confusing. so you don't even have to fetch first event separately with `this.fetchEvent(eventId)`,

getEventParents will return events up to root with eventId included